js/rpg_core/Sprite.js

```diff
@@ -221,11 +225,15 @@ Sprite.prototype.setColorTone = function(tone) {
  * @method _onBitmapLoad
  * @private
  */
-Sprite.prototype._onBitmapLoad = function() {
-    if (this._frame.width === 0 && this._frame.height === 0) {
-        this._frame.width = this._bitmap.width;
-        this._frame.height = this._bitmap.height;
+Sprite.prototype._onBitmapLoad = function(bitmapLoaded) {
+    if(bitmapLoaded === this._bitmap){
+        if (this._refreshFrame && this._bitmap) {
```

Comment:
Nope onBitmapLoad is async. there are some chance `bitmapLoaded !== this._bitmap`